scripts/src/chartrepomanager/chartrepomanager.py

```diff
@@ -218,12 +218,13 @@ def set_package_digest(chart_entry):
         raise Exception("Was unable to compute SHA256 digest, please ensure chart url points to a chart package.")
 
 
-def update_index_and_push(indexfile,indexdir, repository, branch, category, organization, chart, version, chart_url, chart_entry, pr_number, provider_delivery):
+def update_index_and_push(indexfile, indexdir, repository, branch, category, organization, chart, version, chart_url, chart_entry, pr_number, web_catalog_only):
     token = os.environ.get("GITHUB_TOKEN")
     print(f"Downloading {indexfile}")
     r = requests.get(f'https://raw.githubusercontent.com/{repository}/{branch}/{indexfile}')
```

Comment:
Here also we making https request, I think we should include BOT_TOKEN here as well.
I found BOT_TOKEN missing while calling `requests.get` in this module above under function `set_package_digest(chart_entry)`

Reply:
Similar to above the call is not a github api call, call to a chart url.